aospy/test/test_roundtrip.py

```diff
@@ -39,100 +40,100 @@ def model_files_exist(model):
 @unittest.skipIf(not model_files_exist(am2), skip_message)
 class TestAM2(unittest.TestCase):
     def setUp(self):
-        self.olr_test_params = {'proj': aospy_test,
```

Comment:
FYI I like starting things like this on the next line, e.g.
```python
self.olr_test_params = {
    'proj': aospy_test,
    ...
}
```
That way you don't have to change the spacing (and it won't show up on a diff) if you change the name.  Don't worry about here, but for future reference :)